docs/source/index.rst

```diff
@@ -1,25 +1,23 @@
 Welcome to the Livepeer Documentation
 ========================================
 
-`Livepeer`_ is a decentralized video broadcasting platform powered by
-a crypto token on the Ethereum blockchain. Livepeer is for:
+`Livepeer`_ is a decentralized video streaming network build on the Ethereum blockchain. Livepeer is for:
 
 .. _Livepeer: http://livepeer.org
 
 - Developers who want to build applications that include live video.
+- Tokenholders who want to help improve and secure the network by acquiring and staking LPT.
```

Comment:
One of the things I have been thinking about recently is how to get delegators to act more as curators rather than passive capital providers. 

A small step towards such storytelling would be in the docs 

"- Tokenholders who want to help improve, secure and curate the network by acquiring and staking LPT to performant orchestrators"

WDYT ?

Reply:
Fixed in c2055f4

Makes sense to move in that direction language wise. We can also tighten up the language here and in other places as well separately.